cc/mcts_player.h

```diff
@@ -119,64 +129,59 @@ class MctsPlayer {
 
   virtual void NewGame();
 
+  // If visitor is non-null, it is called on each batch of leaves visited during
```

Comment:
I don't understand this comment.

Reply:
That's because it doesn't make any sense :)

It's a comment left over from an earlier attempt at this refactor where I was passing a visitor object around to `SuggestMove`, `ProcessLeaves`, etc instead of having a `SetTreeSearchCallback` method. That was a mess.